### DISCUSSION TITLE
mypy, typing

### DISCUSSION THREAD

**Discussion by peacock0803sz** [sizumita/discord-ext-ui#8] [General]
## 型検査厳しすぎ疑惑

https://github.com/sizumita/discord-ext-ui/blob/6da0d83e8135c2eabe2fa5b8ddee1801eee24102/mypy.ini#L4

`*args` とかにつけてまで`disallow_untyped_defs`すべきか?
https://github.com/sizumita/discord-ext-ui/blob/bc092e50db50c3b8112d40b7bca3a3bf81f6f704/discord/ext/ui/utils.py#L13

## `from __future__ import annotations`と`from typing import List`の混在

上と下はどっちかに寄せるべきでは?

https://github.com/sizumita/discord-ext-ui/blob/6e7131118415f39f6c83058cc9a118c0e4110e6a/discord/ext/ui/types/view_manager.py#L1
https://github.com/sizumita/discord-ext-ui/blob/bc092e50db50c3b8112d40b7bca3a3bf81f6f704/discord/ext/ui/component.py#L1

**Comment by sizumita** [comment]
annotationsなしで行きます
disallow_untyped_defsは正直どっちでもいいんですよね、多分なしでいいとは思いますが...

**Reply by sizumita** [reply]
両方無しにしました
